test/Microsoft.AspNet.Razor.Test/TestFiles/CodeGenerator/CS/Source/EmptyAttributeTagHelpers.cshtml

```diff
@@ -0,0 +1,8 @@
+﻿@addtaghelper "something"
+
+<div>
+    <input type= checked=""class="" />
+    <p age=''>
+        <input type=""checked= class="" />
```

Comment:
it's another invalid case that leads to bogus C#.  might work when `checked` is not bound or bound to a `string` property.  but broken in the context of tag helpers.
not immediately actionable but we need to have a design discussion about errors in tag helper bindings.  probably worth an issue to track that discussion.